backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/workflow/service/impl/AD_Workflow_CopyTemplateCustomizer.java

```diff
@@ -22,44 +22,26 @@
 
 package de.metas.workflow.service.impl;
 
-import de.metas.common.util.time.SystemTime;
 import de.metas.copy_with_details.template.CopyTemplateCustomizer;
 import de.metas.util.InSetPredicate;
 import lombok.NonNull;
 import org.compiere.model.I_AD_WF_Node;
 import org.compiere.model.I_AD_Workflow;
-import org.compiere.model.POInfo;
-import org.compiere.model.copy.ValueToCopy;
+import org.eevolution.model.I_PP_WF_Node_Product;
 import org.springframework.stereotype.Component;
 
-import java.time.format.DateTimeFormatter;
-
 @Component
 public class AD_Workflow_CopyTemplateCustomizer implements CopyTemplateCustomizer
 {
-	private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMdd:HH:mm:ss");
-
 	@Override
 	public String getTableName()
 	{
 		return I_AD_Workflow.Table_Name;
 	}
 
-	@Override
-	public ValueToCopy extractValueToCopy(final POInfo poInfo, final String columnName)
-	{
-		return I_AD_Workflow.COLUMNNAME_Name.equals(columnName) ? ValueToCopy.explicitValueToSet(makeUniqueName()) : ValueToCopy.NOT_SPECIFIED;
-	}
-
 	@Override
 	public @NonNull InSetPredicate<String> getChildTableNames()
 	{
-		return InSetPredicate.only(I_AD_WF_Node.Table_Name);
-	}
-
-	@NonNull
-	private String makeUniqueName()
-	{
-		return I_AD_Workflow.COLUMNNAME_Name.concat("_").concat(DATE_FORMATTER.format(SystemTime.asLocalDateTime()));
+		return InSetPredicate.only(I_AD_WF_Node.Table_Name, I_PP_WF_Node_Product.Table_Name);
```

Comment:
why do we have I_PP_WF_Node_Product.Table_Name here and also in AD_WF_Node_CopyTemplateCustomizer?